packages/app/src/layouts/default.vue

```diff
@@ -98,6 +123,13 @@ const showHeader = computed(() => {
   return currentRoute.meta.header !== false
 })
 
+const showEnableNotificationsBanner = computed(() => {
+  return userProjectStatusStore.cloudStatus === 'allTasksCompleted' &&
+    query.data.value?.localSettings.preferences.desktopNotificationsEnabled === null && (
+    query.data.value?.localSettings.preferences.dismissNotificationBannerUntil ?
+      dayjs().isAfter(dayjs(query.data.value?.localSettings.preferences.dismissNotificationBannerUntil)) : true)
```

Comment:
Is `dayjs` use needed here? I'm assuming we're just storing epoch millis for this value, so couldn't you just do `Date.now() > query.data.value?.localSettings.preferences.dismissNotificationBannerUntil`?

Reply:
We store a time stamp, but I guess I could just do `new Date(query.data.value?.localSettings.preferences.dismissNotificationBannerUntil).getTime()`